pages/sidecar.rst

```diff
@@ -28,15 +28,17 @@ You can get .deb and .rpm packages for Graylog Sidecar in our package repository
 
 Please follow the version matrix to pick the right package:
 
-+-----------------+----------------------------------+
-| Sidecar version | Graylog server version           |
-+=================+==================================+
-| 1.0.x           | 3.0.x,3.1.x,3.2.x                |
-+-----------------+----------------------------------+
-| 0.1.x           | 2.2.x,2.3.x,2.4.x,2.5.x,3.0.x    |
-+-----------------+----------------------------------+
-| 0.0.9           | 2.1.x                            |
-+-----------------+----------------------------------+
++-------------------+-----------------------------------+
+| Sidecar version   | Graylog server version            |
++===================+===================================+
+| 1.1.x             | 3.2.5 or higher, 4.0 or higher    |
```

Comment:
Graylog 3.3.x also supports the new beats collectors.
I'm not sure how explicit we should be here.
I'd say `3.2.5 or higher` covers everything

Reply:
Good point!